### PR TITLE
Added suspend filter for customers #6685

### PR DESCRIPTION
**Pull request by papnoisanjeev**
## Issue Reference
#6685

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@papnoisanjeev 
There should be a **difference** between the **applied filter**, if the admin is filtering **suspended / not** sus[ended customer, please find the screenshot [Here](https://prnt.sc/nTNvwZ2m20Fr)

**Comment by papnoisanjeev** [comment]
@Sanjaybhattwebkul 

Please check again as checked with same commit able to see applied filter here https://prnt.sc/boC5HN9L2On4

**Comment by papnoisanjeev** [comment]
> @Sanjaybhattwebkul
> 
> Please check again as checked with same commit and able to see applied filter here https://prnt.sc/boC5HN9L2On4

**Review by Sanjaybhattwebkul — DISMISSED** [review]

File: packages/Webkul/Admin/src/DataGrids/CustomerDataGrid.php [hunk 1]
```diff
@@ -158,6 +159,17 @@ public function addColumns()
                 return $html;
             },
         ]);
+
+        $this->addColumn([
+            'index'       => 'is_suspended',
+            'label'       => trans('admin::app.customers.customers.suspended'),
+            'type'        => 'boolean',
+            'filter-type' => 'suspended',
```
Comment (devansh-webkul) [inline packages/Webkul/Admin/src/DataGrids/CustomerDataGrid.php:167]: Any use of this key?

Comment (devansh-webkul) [reply packages/Webkul/Admin/src/DataGrids/CustomerDataGrid.php:167]: filter-type

Comment (papnoisanjeev) [reply packages/Webkul/Admin/src/DataGrids/CustomerDataGrid.php:167]: We need this suspended key for **filters** only not for data grid so using this key we have removed this from data grid and only added into filter.

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**